PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/ClickToMove.rbxmx

```diff
@@ -1268,19 +1235,29 @@ local function CreateClickToMoveModule()
 			return CFrame.new(newCamPos, newCamPos + flippedLook)
 		end
 
+		local v3 = Vector3.new
```

Comment:
why do we do this here instead of at the top of the file and use this for all Vector3.new references? Also the names here are a little short